scripts/shared/clusters.sh

```diff
@@ -106,7 +106,9 @@ function create_kind_cluster() {
 }
 
 function deploy_cni() {
-    [[ -n "${cluster_cni[$cluster]}" ]] || return 0
+    if [[ "${cluster_cni[$cluster]}" == "ovn" ]] || [[ -z "${cluster_cni[$cluster]}" ]]; then
```

Comment:
Alternatively, you can add a `deploy_ovn_cni` function which could do nothing (or print some message stating its already deployed).
This way if in the future we can deploy ovn like that, you won't need to touch this line again.

Reply:
That was my initial code, wasn't sure which of two worked better. Went with this to avoid a function that is basically a no-op.